src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -175,16 +173,18 @@ export class zonePacketHandlers {
         serverTime: Int64String(server.getServerTime()),
         serverTime2: Int64String(server.getServerTime()),
       });
-
+      
       server.sendData(client, "Command.ItemDefinitions", {
         // sends full list of item definitions
         data: {
-          itemDefinitions: itemDefinitions.map((itemDef: any) => {
+          itemDefinitions: Object.keys(server._itemDefinitions).map((itemDefId: any) => {
```

Comment:
since itemDefinitions never changes, we should create it `Object.keys(server._itemDefinitions).map((itemDefId: any)` only once and assign it in a const